### DISCUSSION TITLE
Matrix mistakenly deployed

### DISCUSSION THREAD

**Discussion by Toflamus** [alshedivat/al-folio#2375] [Q&A] [answered]
The original code:

\begin{equation}
    \begin{bmatrix}
    z[s+1]\\
    \vec i_r[s+1]\\
    h[s+1]\\
    \end{bmatrix}
    =
    \begin{bmatrix}
    1 & 0 & 0\\
    0 & \mathbb A_{RC} & 0 \\
    0 & 0 & A_H\\
    \end{bmatrix}
    \begin{bmatrix}
    z[s]\\
    \vec i_r[s]\\
    h[s]\\
    \end{bmatrix}
    + 
    \begin{bmatrix}
    -\frac{\eta [s] \Delta t}{Q} & 0\\
    \vec b_{RC} & 0\\
    0 & A_H-1
    \end{bmatrix}
    \begin{bmatrix}
    I & sgn(I)\\
    \end{bmatrix}
\end{equation}
However, it ends up to be sth like this:

<img width="704" alt="Wrongmatrix" src="https://github.com/alshedivat/al-folio/assets/112740603/353b3af1-c568-46e2-bfc5-82904812ba1c">

While it looks good in my .md file previewer
<img width="550" alt="Goodmatrix" src="https://github.com/alshedivat/al-folio/assets/112740603/6c457c68-f49d-461c-959a-1537623238bd">

Anybody can help thank you!

**Comment by george-gca** [comment]
We use [MathJax](https://www.mathjax.org/) to render LaTeX code. If it doesn't handle your code well, I am afraid there is not much we can do. You can do some test in their [live demo site](https://www.mathjax.org/#demo).

**Reply by Toflamus** [reply]
Thanks! However, I tried several other times and found my code works perfectly fine on the MathJax website but just fail to work on the blog.

**Comment by alshedivat** [comment, marked as answer]
try using `\\` instead of `\` for line breaks in the matrices:
```latex
\begin{equation}
\begin{bmatrix}
z[s+1] \\
\vec i_r[s+1] \\
h[s+1]
\end{bmatrix}
=
\begin{bmatrix}
1 & 0 & 0 \\
0 & \mathbb A_{RC} & 0 \\
0 & 0 & A_H
\end{bmatrix}
\begin{bmatrix}
z[s] \\
\vec i_r[s] \\
h[s]
\end{bmatrix}
+
\begin{bmatrix}
-\frac{\eta [s] \Delta t}{Q} & 0 \\
\vec b_{RC} & 0 \\
0 & A_H-1
\end{bmatrix}
\begin{bmatrix}
I & sgn(I)
\end{bmatrix}
\end{equation}
```

**Reply by Toflamus** [reply]
Thanks! Actually, the original code is `\\`. I do not know why, but after copying and pasting it becomes a `\`. Thank you for your help!
